doc/api/dns.md

```diff
@@ -568,8 +568,10 @@ dns.setServers([
 
 An error will be thrown if an invalid address is provided.
 
-The `dns.setServers()` method must not be called while a DNS query is in
-progress.
+The `dns.setServers()` method affects only the methods of [`dns.Resolver`][]
+that have equivalents as `dns.*()` APIs (and specifically *not*
+[`dns.lookup()`][]). It must not be called while those methods are in progress.
```

Comment:
This is confusing to me – it does not affect the `dns.Resolver` methods, but their `dns.*` equivalents, imo, so maybe the text here should be reversed?

Also, calling `dns.setServers()` should always be okay in recent versions of Node. Calling `resolver.setServers()` is also okay, but will yield an error if there are active queries.

Reply:
As I understand that, it affects methods that are accessible both on dns.Resolver and on dns.

Reply:
@szmarczak `dns.Resolver` instances have their own server settings, so the global `dns.setServers()` should not affect them?

Reply:
I made it explicit.